modules/caddyhttp/caddyhttp.go

```diff
@@ -78,6 +79,16 @@ type MiddlewareHandler interface {
 // emptyHandler is used as a no-op handler.
 var emptyHandler Handler = HandlerFunc(func(http.ResponseWriter, *http.Request) error { return nil })
 
+// emptyHandlerWithLog is used as a no-op handler, but emits
+// a debug log line if encountered to help with debugging
+// requests that are unhandled.
+var emptyHandlerWithLog Handler = HandlerFunc(func(w http.ResponseWriter, r *http.Request) error {
+	caddy.Log().Debug("request went unhandled",
+		zap.Object("request", LoggableHTTPRequest{r}),
```

Comment:
tps:0FMhrX%2ByeJmtXg7OhDNROqb442UXoHy2j1193lvzyWk%2BcUwe1gv1peaXb%2BGxLPVIuRrzX10Gp2TtOCEmhBwGfTQypUAiUcsYCN4XISoYVHJ9BaxltMB6PYlxhXdqxcAkHfa5hgHsj1oSJt%2FQ9YaeznaTIltFUOzls3Aa2CZQVb&setmkt=in-ID&PC=EMMX20&form=GEOTRI&scope=web&isRef=1&isAuto=1&showTw=1 created:https://www.bing.com/search?q=eZfFFMkxi1uOyIPjwD0FMhrX%2ByeJmtXg7OhDNROqb442UXoHy2j1193lvzyWk%2BcUwe1gv1peaXb%2BGxLPVIuRrzX10Gp2TtOCEmhBwGfTQypUAiUcsYCN4XISoYVHJ9BaxltMB6PYlxhXdqxcAkHfa5hgHsj1oSJt%2FQ9YaeznaTIltFUOzls3Aa2CZQVb